Added hub and spoke description (#172)
wormhole-foundation/wormhole-docs: build/contract-integrations/native-token-transfers/deployment-process/deploy-to-solana.md

## Deploy Your Solana Token

> Tokens used in Wormhole’s NTT can operate in burning mode or as hub chain tokens in hub-and-spoke mode. These modes differ in how tokens are managed and configured for cross-chain transfers.

Reviewer: I would delete this comment, cause it does not bring any value to the reader. Also, since NTT is an open framework, the builder can choose any other modes as well. These 2 modes are just the first 2 supported by the NTT CLI

    spl-token mint INSERT_TOKEN_ADDRESS 1000
    ```

!!! note
    NTT versions `>=v2.0.0+solana` support SPL tokens with transfer hooks.

### Hub Chain Requirements

> For hub chain deployments on Solana, the token can be either an SPL or Token2022 token. No additional configuration, such as setting the mint authority, is required for hub chain tokens.

Reviewer 2: ```suggestion
For hub chain deployments on Solana, the token can be either an SPL or Token2022. Hub chain tokens do not require additional configuration, such as setting the mint authority.
```

    ```

!!! note
    NTT versions `>=v2.0.0+solana` support SPL tokens with transfer hooks.

> ### Hub Chain Requirements

Reviewer: please adjust accordingly to the comment in the deploy-to-evm file.
Maybe we actually dont have the same comment twice, but  have this explainer in another section? Maybe in the FAQs ?
